manifests/ppa.pp

```diff
@@ -74,7 +74,7 @@
     }
 
     $_proxy = $apt::_proxy
-    if $_proxy['host'] {
+    if $_proxy != undef and $_proxy['host'] {
```

Comment:
I don't understand how `$_proxy` can be undef. It's set in init.pp:
https://github.com/puppetlabs/puppetlabs-apt/blob/e0b3a5db6abb043f106614dffe341f68d88158ab/manifests/init.pp#L252
So I would assume it cannot be undef. Even if `$apt::proxy_defaults` and `$proxy` are undef, it would still contain `{ 'perhost' => $_perhost }`